guides/source/documents.yaml

```diff
@@ -140,6 +140,11 @@
       work_in_progress: true
       url: api_app.html
       description: This guide explains how to effectively use Rails to develop a JSON API application.
+    -
+      name: Active Record and PostgreSQL
+      work_in_progress: true
+      url: active_record_postgresql.html
+      description: This guide covers PostgreSQL specific usage of Active Record.
```

Comment:
I had discussed it with @senny and he rightly mentioned that this guide is not in a stage to be added to guides list as it just lists points. There is no prose like other guides.